TestApp/ios/Podfile.lock

```diff
@@ -288,13 +290,14 @@ DEPENDENCIES:
   - "RNCAsyncStorage (from `../node_modules/@react-native-community/async-storage`)"
   - RNFS (from `../node_modules/react-native-fs`)
   - RNGestureHandler (from `../node_modules/react-native-gesture-handler`)
+  - RNScreens (from `../node_modules/react-native-screens`)
   - Yoga (from `../node_modules/react-native/ReactCommon/yoga`)
 
 SPEC REPOS:
-  https://cdn.cocoapods.org/:
-    - AppCenter
   https://github.com/CocoaPods/Specs.git:
     - boost-for-react-native
+  trunk:
```

Comment:
curious about that change moving away from the CDN

Reply:
`trunk` is a different name for the same main spec repo backed by a CDN, please see https://github.com/CocoaPods/CocoaPods/issues/9190#issuecomment-535330760:

> trunk is the new name for the main specs repo which is backed by a CDN. This repo is included by default, and since your Podfile does not have any custom spec repos specified, it is being included in this case.